### PR TITLE
Add enable command

### PR DESCRIPTION
**Pull request by kevinliuyifeng**
Add enable command.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Major release (change is NOT backward compatible with prior release)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Then, update your project's pom.xml file dependencies, as follows:
   <dependency>
       <groupId>com.yahoo.imapnio</groupId>
       <artifactId>imapnio.core</artifactId>
-      <version>3.0.7</version>
+      <version>3.0.8</version>
```
Comment (jui8wang) [inline README.md]: please change README.md  line 45 to 3.0.8 too

File: core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    @Override
+    public ByteBuf getCommandLineBytes() throws ImapAsyncClientException {
+        final ByteBuf sb = Unpooled.buffer(ENABLE_BUF_LEN);
+
+        sb.writeBytes(ENABLE_SP_B);
+
+        for (int i = 0, len = capabilities.length; i < len; i++) {
+            sb.writeBytes(capabilities[i].getBytes(StandardCharsets.US_ASCII));
```
Comment (jui8wang) [inline core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java]: have a comment as following
// capability ABNF is:
// capability      = ("AUTH=" auth-type) / atom

File: README.md (hunk 1 above)
Comment (kevinliuyifeng) [reply README.md]: This is README.md

File: core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        capabilities = null;
+    }
+
+    @Override
+    public ByteBuf getCommandLineBytes() throws ImapAsyncClientException {
+        final ByteBuf sb = Unpooled.buffer(ENABLE_BUF_LEN);
+
+        sb.writeBytes(ENABLE_SP_B);
+
+        for (int i = 0, len = capabilities.length; i < len; i++) {
```
Comment (nvsnvikram) [inline core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java]: Remove len and use capabilities.length

File: core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            // capability ABNF is:
+            // capability = ("AUTH=" auth-type) / atom
+            sb.writeBytes(capabilities[i].getBytes(StandardCharsets.US_ASCII));
+            if (i < len - 1) { // do not add space for last item
```
Comment (nvsnvikram) [inline core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java]: Remove space on ENABLE and that way you can remove this if check as well.

File: core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    @Override
+    public ImapCommandType getCommandType() {
+        return ImapCommandType.ENABLE;
+    }
+
+    /** Byte array for CR and LF, keeping the array local so it cannot be modified by others. */
+    private static final byte[] CRLF_B = { '\r', '\n' };
+
+    /** Enable and space. */
+    private static final byte[] ENABLE_SP_B = "ENABLE".getBytes(StandardCharsets.US_ASCII);
```
Comment (nvsnvikram) [inline core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java]: ENABLE_SP_B -> ENABLE_B

**Pull request merged**